elesto-dao/elesto
# feat: add stricter validation of did document

### Description

- cherry-pick commits from DID audits into main branch

### How to test

- see commits on => https://github.com/elesto-dao/elesto/commits/release/v1.0.0
- ensure code and commits match
